apps/demos/Demos/Stepper/Overview/jQuery/index.js

```diff
@@ -0,0 +1,76 @@
+$(() => {
+  const initialStepperSettings = {
+    rtlEnabled: false,
+    selectedIndex: 2,
+    selectOnFocus: false,
+    orientation: orientations[0].value,
+    linear: navigationModes[0].value,
+  };
+
+  const stepperWithIcons = $('#withText').dxStepper({
```

Comment:
For each stepper, aria-labelledby attribute should be presented to link them with their labels. See StepTemplate demo.